src/studio/src/designer/DataModeling.Tests/Json/SeresConverterTests.cs

```diff
@@ -135,6 +135,30 @@ await using (var xw = XmlWriter.Create(sw, new XmlWriterSettings { Indent = true
             XmlSchemaAssertions.IsEquivalentTo(expectedXsd, actualXsd);
         }
 
+        [Theory]
+        [InlineData(@"Model\JsonSchema\SeresSimpleContentRestriction.json", @"Model\XmlSchema\SeresSimpleContentRestriction.fromJson.xsd")]
```

Comment:
Is using `fromJson` a new naming convention?

Reply:
It is used where the xsd -> json schema -> xsd conversion will produce a slightly different xsd (but functionally the same)